src/components/tutorialSelector/index.tsx

```diff
@@ -105,7 +112,8 @@ const TutorialSelector: React.FC = (): JSX.Element => {
           <SingleTutorial
             key={tutorialName + index.toString()}
             onClick={(): void => {
-              setTutorialStep(1);
+              setTutorialStep(tutStep);
+              localStorage.setItem("tutName", value.tutorial_title);
               setTutorialName(value.tutorial_title);
               setSelectorDisplay(false);
               setTutorialDisplay(true);
```

Comment:
From my PR that Clay reviewed, I learned that inline functions incur a performance cost in React. If you could move this inline function into a defined function (for example `const selectTutorial = () => {}`) that would be great.